website/docs/configuring_clients.md

```diff
@@ -344,6 +344,238 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 If you created the application selecting _Storyboard_ as _User Interface (see [Create a new project](#create-a-new-project)), we are done: the application now works on _iOS 12_.
 Otherwise you will have to manually create a storyboard.
 ## Webpush
- - AEROGEAR-10134
+
+Prerequisites: to be able to follow the instructions below, you must have [Node.js](https://nodejs.org/) installed and 
+working and you must have a [WebPush Variant](configuring_variants#WebPush) already configured.
+
+The guide below will be a simple step-by-step guide to integrate a sample `React` application with _UnifiedPush Server_.
+
+### Create a new project
+This guide will walk you through the creation of a _typescrypt React_ application.
+To create one, run:
+```bash
+$ npx create-react-app hello-world-webpush --template typescript
+```
+
+### Add the required dependencies
+```bash
+$ cd hello-world-webpush
+$ npm install --save @aerogear/push
+```
+
+### Add the service worker
+
+To be able to handle _push notifications_, a service worker is needed. In this guide we will provide a very simple implementation,
+for instructions about writing service workers please refer to the [official documentation](https://developer.mozilla.org/en-US/docs/Web/API/Push_API). 
+
+Inside the `public` folder, create a file named `simple-sw.js` with the following content:
+```javascript
+self.addEventListener('push', event => {
+  const data = event.data.text();
+  const msg_chan = new MessageChannel();
+  clients.matchAll().then(clients => {
```

Comment:
The first clients should probably be `self.clients` and the clients variable in the closure should be renamed so it doesn't conflict the with name of self.clients.  (If my eyeballing service worker code is right).